src/lib/addons/webhook.ts

```diff
@@ -94,6 +103,13 @@ export default class Webhook extends Addon {
             state = 'failed';
             const failedMessage = `Webhook request failed with status code: ${res.status}.`;
             stateDetails.push(failedMessage);
+            if (this.flagResolver.isEnabled('addonUsageMetrics')) {
+                this.eventBus.emit(ADDON_EVENTS_HANDLED, {
+                    result: state,
+                    destination: 'webhook',
```

Comment:
Probably makes sense but intentional that this is `webhook` and not `serviceName`?

Reply:
Yeah we'll add known ones to a static list of known service names later and cross check against that so we don't fill metrics up with random stuff